services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/AuthorizationManagerImpl.java

```diff
@@ -608,4 +609,30 @@ public AuthorizationStatus canAccessMembershipInvitationSubmission(UserInfo user
 		}
 		return AuthorizationManagerUtil.accessDenied("Unauthorized to access membership invitation " + mis.getId() + " for " + accessType);
 	}
+
+	@Override
+	public AuthorizationStatus canAccessMembershipInvitationSubmission(String misId, MembershipInvtnSignedToken token, ACCESS_TYPE accessType) {
+		try {

```

Comment:
omit 'misId' since the ID is already in the token.

Reply:
Fixed